src/file-2.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Has Tag ([xoo:HasTag](http://127.0.0.1:4000/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:4000/project/issues?id=sample&pullRequest=6726&issues=AWk46nlf6wjeuPDY-rN0&open=AWk46nlf6wjeuPDY-rN0)

[ISSUE_KEY]: AWk46nlf6wjeuPDY-rN0